src/main/java/module/training/ui/comp/FutureSettingPanel.java

```diff
@@ -77,7 +77,11 @@ private void jbInit() {
         intensity = new IntensityComboBox(0);
         intensity.setSelectedIndex(firstFutureTraining.getTrainingIntensity());
         staminaTrainingPart = new IntensityComboBox(10);
-        staminaTrainingPart.setSelectedIndex(firstFutureTraining.getStaminaPart() - 10);
+        int staminaPart = firstFutureTraining.getStaminaPart();
+        if (staminaPart - 10 < 0){
```

Comment:
why not simply `if (staminaPart < 10) {staminaPart = 10;}`

Reply:
it is only logical approach :D